both/access_control.js

```diff
@@ -231,101 +229,75 @@ if(Meteor.isServer) {
         'createdAt',
       ];
 
-      if(_.difference(fields, allowedFields).length > 0) {
+      return _onlyAllowedFields(fields, allowedFields);
```

Comment:
I'm used to the private functions start with an underscore convention from python. It doesn't seem necessary here, as `_onlyAllowedFields` isn't a method on a class, right?